package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "foreman_bootdisk",
+  "version": "1.0.0",
+  "description": "Plugin for Foreman that creates iPXE-based boot disks to provision hosts without the need for PXE infrastructure.",
+  "main": "index.js",
+  "scripts": {
+    "lint": "tfm-lint --plugin -d /webpack",
+    "test": "tfm-test --plugin",
+    "test:watch": "tfm-test --plugin --watchAll",
+    "test:current": "tfm-test --plugin --watch",
+    "publish-coverage": "tfm-publish-coverage",
+    "stories": "tfm-stories --plugin",
+    "stories:build": "tfm-build-stories --plugin",
+    "create-react-component": "yo react-domain"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/theforeman/foreman_bootdisk.git"
+  },
+  "bugs": {
+    "url": "http://projects.theforeman.org/projects/foreman_bootdisk/issues"
+  },
+  "peerDependencies": {
+    "@theforeman/vendor": ">= 8.8.0"
+  },
+  "dependencies": {
+    "react-intl": "^2.8.0"
+  },
```

Comment:
`react-intl` has moved to foreman-js in v8.15.0 so you can remove the whole `dependencies` section